### PR TITLE
Adding the Service and Endpoint retrieval methods in the common lib

### PR DESCRIPTION
**Pull request by nandajavarma**

### CODE REVIEW

File: pkg/kube/services.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+func (kc *KubernetesClient) GetService(svcName string,
+	namespace string) (*corev1.Service, error) {
+	service, err := kc.Client.CoreV1().Services(namespace).Get(context.TODO(),
+		svcName, metav1.GetOptions{})
+
+	return service, err
+}
+
+// nolint:typecheck
+func (kc *KubernetesClient) GetEndpoints(service *corev1.Service,
```
Comment (angelbarrera92) [inline pkg/kube/services.go:24]: Consider passing the context as an argument in the function to use it in the `Get` function (when using the kubeclient).

File: pkg/kube/services.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// nolint:typecheck
+func (kc *KubernetesClient) GetService(svcName string,
```
Comment (angelbarrera92) [inline pkg/kube/services.go:15]: Consider passing the context as an argument in the function to use it in the `Get` function (when using the kubeclient).

File: .rules/.golangci.yml [hunk 3]
```diff
@@ -53,7 +54,6 @@ linters:
     - nlreturn
     - nolintlint
     - prealloc
-    - scopelint
```
Comment (angelbarrera92) [inline .rules/.golangci.yml:56]: What is/are the reasons to remove `scopelint` and add `exportloopref`? Does it make sense to also push these changes to the template?

**Review by angelbarrera92 — CHANGES_REQUESTED** [review]
I think passing the context to a shared library is worthy for the consumers.

File: .rules/.golangci.yml (hunk 3 above)
Comment (nandajavarma) [reply .rules/.golangci.yml:56]: scopelint is deprecated and replaced by exportloopref.

Comment (angelbarrera92) [reply .rules/.golangci.yml:56]: So we can push this change to the template? :)

File: pkg/kube/services.go (hunk 2 above)
Comment (nandajavarma) [reply pkg/kube/services.go]: fixed

File: .rules/.golangci.yml (hunk 3 above)
Comment (nandajavarma) [reply .rules/.golangci.yml:56]: opening a PR !

**Review by angelbarrera92 — APPROVED** [review]
LGTM!

**Pull request merged**